script.js

```diff
@@ -1,69 +1,101 @@
-function computerPlay() {
-  const choice = ['rock','paper','scissors'];
-  return choice[Math.floor(Math.random()*choice.length)]
+const choices = ['rock', 'paper', 'scissors'];
+
+function randomInteger(max, min = 0) {
```

Comment:
add general utility function to generate a random integer within range

Reply:
Do you mean, should I create a new function to generate a number integer and then call it?